src/Microsoft.TemplateEngine.Abstractions/ICacheParameter.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.TemplateEngine.Abstractions
 {
     /// <summary>
     /// Defines the representation of a parameter in template cache.
-    /// This doesn't include parameters with choices which are represented by <see cref="ICacheTag"/>.
+    /// In Orchestrator.RunnableProjects (template.json) parameter symbols are cached (all types except 'choice'). Choice parameters are stored as <see cref="ICacheTag"/>in <see cref="ITemplateInfo.Tags"/> collection.
```

Comment:
This comment is describing the implementation of another type. When the implementation changes, this will need to change. I believe it is best if we put this kind of information to our github wiki page.

Reply:
I added it here due to usually you use `ICacheParameter` and not `CacheParameter`; and it's important to know the difference. However, I might remove you insist on it.